scripts/leaderboard_backend/eval.py

```diff
@@ -0,0 +1,14 @@
+# type: ignore
+import subprocess
+
+from database_helper import DatabaseHelper
+
+
+def eval_model(model_name: str, precision: str, num_fewshot: str):
+    subprocess.run(["./eval_script.sh", model_name, precision, num_fewshot])
```

Comment:
Probably, you can call evaluation script directly (import eval func and call it with necessary parameters)

Reply:
Fixed